spec/controllers/application_templates_controller_spec.rb

```diff
@@ -120,7 +121,7 @@
       end
       it 'redirects back' do
         submit
-        expect(response).to redirect_to :back
+        expect(response).to redirect_to 'http://test.host/redirect'
```

Comment:
Interesting. Is this test accurately named now?  It seems like it's redirecting *somewhere*, but is it "back"?